py_zipkin/_encoding_helpers.py

```diff
@@ -409,6 +455,48 @@ def encode_span(self, span_builder):
 
         return encoded_span
 
-    def encode_queue(self, queue):
-        """Concatenates the list to a JSON list"""
-        return '[' + ','.join(queue) + ']'
+
+class _V2JSONEncoder(_BaseJSONEncoder):
+    """JSON encoder for V2 spans."""
+
+    def encode_span(self, span_builder):
+        """Encodes a single span to JSON."""
+        span = span_builder.build_v2_span()
+
+        json_span = {
+            'traceId': span.trace_id,
+            'name': span.name,
```

Comment:
name is optional in v2, and shouldn't be set unless you actually know it (ex don't serialize "unknown"). In v1 we had to hack with empty string when unknown

Reply:
Even in V1 JSON I should use `""`?